### PR TITLE
Add cmdlet help examples for RunCommand v2 (Set, Get) cmdlets

### PR DESCRIPTION
**Pull request by viveklingaiah**
## Description

Adding cmdlet help examples for Set-AzVMRunCommand, Set-AzVmssVMRunCommand, Get-AzVMRunCommand, Get-AzVmssVMRunCommand cmdlets. Targeting this for November 2022 Powershell release. I may edited too many files. I just need examples to show up when you do for ex. "Get-Help Set-AzVMRunCommand -examples"

## Checklist

- [Yes ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by grizzlytheodore — APPROVED** [review]

**Comment by grizzlytheodore** [comment]
@vivlingaiah  please remove any updates for Set- Cmdlets from this PR !

**Comment by grizzlytheodore** [comment]
@Nickcandy to update the md files for autorest generated, which files should we be updating in this PR? anything to change in this PR?

File: src/Compute/Compute.Autorest/docs/Get-AzVmssVMRunCommand.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+EndTime          : 10/27/2022 9:10:55 PM
+Statuses         :
+```
+
+Get a Run Command for VM with Instance View. Instance View contains execution state of run command (Succeeded, Failed, etc.), exit code, standard output and standard error generated by executing the script using Run Command. A non-zero ExitCode indicates an unsuccessful execution.
+
+### Example 3: Get all Run Commands for a VMSS VM instance
+
+```powershell
+Get-AzVmssVMRunCommand -ResourceGroupName MyRG -VMName MyVM
```
Comment (Nickcandy) [inline src/Compute/Compute.Autorest/docs/Get-AzVmssVMRunCommand.md:39]: Please check with if this cmdlet have the parameter VMName

**Comment by grizzlytheodore** [comment]
@Nickcandy  looks all good. Please review/merge! thank you

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**